Towny/src/main/java/com/palmergames/bukkit/towny/TownyFormatter.java

```diff
@@ -1046,12 +1047,14 @@ public static String getTime() {
 	
 	private static String formatWebUrl(SpawnLocation spawnLocation) {
 		String webUrl = "";
-		if (TownySettings.isUsingWebMapStatusScreens() && spawnLocation.hasSpawn() && !TownySettings.getWebMapUrl().isEmpty())
+		if (TownySettings.isUsingWebMapStatusScreens() && spawnLocation.hasSpawn() && !TownySettings.getWebMapUrl().isEmpty()) {
+			World world = spawnLocation.getSpawnOrNull().getWorld();
+			String worldName = TownySettings.isUsingWorldKeyForWorldName() ? world.getKey().toString() : world.getName();

```

Comment:
We still support 1.16 (for now) and spigot's World class didn't implement Keyed yet back then, it was only added in 1.18.2 while paper already had it

```suggestion
			String worldName = TownySettings.isUsingWorldKeyForWorldName() && ( PaperLib.isPaper() || MinecraftVersion.isNewerThanOrEquals(MinecraftVersion.1_18_2)) ? world.getKey().toString() : world.getName();

```